### DISCUSSION TITLE
What happened to the CDN and GitHub link on the main site ractivejs.org?

### DISCUSSION THREAD

**Discussion by donnyv** [ractivejs/ractive#3399] [General]
I went to the site and noticed there is no link to the CDN or the Github release page or repo.

**Comment by ceremcem** [comment]
Ractivejs.org had been replaced by ractive.js.org long time ago. You may find additional information within the issues.

**Reply by donnyv** [reply]
I see, I never even noticed. This time to get to the site I did a Google search and must of have clicked on the wrong link. Do you guys have access to that old site? If so, you should probably setup a redirect.

**Reply by evs-chris** [reply]
We switched over to the js.org site so no one have to maintain the domain registration, and I thought the old site would eventually go away. I guess it never really did. I'm not sure where that site lives exactly, as it predates me by a bit.

**Comment by ceremcem** [comment]
Fixing the Google links would be a good idea. Isn't there another way to
inform Google to discard all of the cache from the former domain?
